src/components/new-request/search.vue

```diff
@@ -239,7 +239,7 @@
         </template>
 
         <!-- Numbered company bullets -->
-        <template v-if="selectedCompanyType === CompanyType.NUMBERED_COMPANY || isFederal" cols="12">
+        <template v-if="selectedCompanyType === CompanyType.NUMBERED_COMPANY" cols="12">
```

Comment:
I remove `isFederal `check because it shows bullet point for Numbered company along with text if Federal jurisdiction is selected.

![image](https://github.com/bcgov/namerequest/assets/116035339/78e9362e-40cd-42e3-8adb-9d58d478ac24)

Reply:
Only one set of bullets should display in either case.

It seems to work correctly in the temp URL. Test cases:
1. change name of a XCP with jurisdiction=Federal
2. restore a XCP with jurisdiction=Federal
3. xpro registration of a XCP with jurisdiction=Federal

Reply:
Ketaki, this change will break the if federal is selected in Restoration flow:
![federal restore flow error](https://github.com/bcgov/namerequest/assets/122301442/716b413a-f267-4cb2-b370-2e8b4735f9b9)

In my temp URL, there's usually the go to Colin button:
![go to colin in my temp url](https://github.com/bcgov/namerequest/assets/122301442/7be3a2a4-cdd4-41d5-93fd-01b715e49498)

Reply:
okay. Then I think we need to look at the conditions for showing colin button. If we add conditional at the the template level it breaks bullets too. What do yo uthink?

Reply:
Hmmm, good point. I think we need to have a special condition for restorations maybe? That might make things messy though.